script/component-migration/transformers/additionalinfo.js

```diff
@@ -0,0 +1,20 @@
+function additionalInfoReplacement() {
+  const removeClassName = (componentString, _propName) => {
+    let noClass = componentString;
+
+    noClass = noClass.replace(/className=["{](.)+?["}]\s/ms, '');
```

Comment:
The regex here is being a bit too aggressive and is removing `className` from some child nodes:

<details>
<summary>Script error</summary>

![image](https://user-images.githubusercontent.com/2008881/145866528-161a2332-df99-4c4f-b421-a87d78a4717d.png)

</details>

<details>
<summary>Showing error in diff</summary>

![image](https://user-images.githubusercontent.com/2008881/145866623-4e5c0b7c-6a60-49fd-91c3-b17ebaafd5aa.png)

</details>

We could either:

- improve the regex, or
- not do anything with `className`


Since `classNames` is used on `<AdditionalInfo>` (even though it's only once), I think we should try to fix the regex.

![image](https://user-images.githubusercontent.com/2008881/145867009-ac19f296-d8ed-4b30-b32e-b10a44088d9d.png)

This suggestion is the same kind of [change I made in the core script](https://github.com/department-of-veterans-affairs/vets-website/pull/19646#discussion_r766102215) to fail the match if we find a closing tag. This works because we only want to replace a `className` that comes before that first `>` bracket.

```suggestion
    noClass = noClass.replace(/(?!.*>)className=["{](.)+?["}]\s/ms, '');
```

Reply:
I've updated the regex, but it's still not working properly. Your suggestion didn't work. :(